community_tasks/arabic_evals.py

```diff
@@ -196,7 +198,367 @@ def arabic_exams(line, task_name: str = None):
     )
 
 
-_TASKS = ARABIC_MMLU_TASKS + ACVA_TASKS + [arabic_exams_task]
+# ALGHAFA NATIVE ##
+# fmt: off
+ALGHAFA_SUBSETS = [
+    "mcq_exams_test_ar", "meta_ar_dialects", "meta_ar_msa", "multiple_choice_facts_truefalse_balanced_task", "multiple_choice_grounded_statement_soqal_task",
+    "multiple_choice_grounded_statement_xglue_mlqa_task", "multiple_choice_rating_sentiment_no_neutral_task", "multiple_choice_rating_sentiment_task",
+    "multiple_choice_sentiment_task"
+]
+# fmt: on
+
+
+class CustomALGHAFATask(LightevalTaskConfig):
```

Comment:
```suggestion
class CustomAlGhafaTask(LightevalTaskConfig):
```
Please use the standard camel casing for classes. You'll need to edit where it's called too.